tests/aws/services/ec2/test_ec2.py

```diff
@@ -694,6 +694,84 @@ def _create_security_group() -> dict:
         assert e.value.response["ResponseMetadata"]["HTTPStatusCode"] == 400
         assert e.value.response["Error"]["Code"] == "InvalidSecurityGroupId.DuplicateCustomId"
 
+    @markers.snapshot.skip_snapshot_verify(
+        paths=[
+            "$..Tags",
+            "$..SecurityGroupForVpcs..Description",
+            "$..SecurityGroupForVpcs..GroupId",
+            "$..SecurityGroupForVpcs..GroupName",
+        ]
+    )
```

Comment:
Generally paths are skipped only if they are not implemented in LocalStack. Why are these skipped?